lighthouse-core/lib/page-functions.js

```diff
@@ -127,13 +123,16 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
   // TODO(paulirish): Don't clean title attribute from all elements if it's unnecessary
   const autoFillIgnoreAttrs = ['autofill-information', 'autofill-prediction', 'title'];
 
-  try {
-    // ShadowRoots are sometimes passed in; use their hosts' outerHTML.
-    if (element instanceof ShadowRoot) {
-      element = element.host;
-    }
+  // ShadowRoots are sometimes passed in; use their hosts' outerHTML.
+  if (element instanceof ShadowRoot) {
+    element = element.host;
+  }
 
+  try {
+    /** @type {Element} */
+    // @ts-expect-error - clone will be same type as element - see https://github.com/microsoft/TypeScript/issues/283
     const clone = element.cloneNode();
```

Comment:
`clone` _is_ an `Element` since `element` is an `Element`, but the tsc dom types don't reflect this. When the mentioned issue (https://github.com/microsoft/TypeScript/issues/283) is fixed (a fix was tried last year but backed out when it broke some stuff), this `@ts-expect-error` will start erroring and can be removed